Modules/MFT/src/QcMFTDigitCheck.cxx

```diff
@@ -105,8 +119,73 @@ Quality QcMFTDigitCheck::check(std::map<std::string, std::shared_ptr<MonitorObje
 
 std::string QcMFTDigitCheck::getAcceptedType() { return "TH1"; }
 
+void QcMFTDigitCheck::readMaskedChips()
+{
+  o2::ccdb::CcdbApi api;
+  api.init(mDeadMapCcdbAddress.data());
+  long timestamp = -1;
```

Comment:
If you are going to use this check in async processing, this might request a wrong object, not matching the time of data.

Reply:
Is there a standardised way of accessing CCDBB objects with the validity corresponding to the data?

Reply:
Not well exercised, but... `UserCodeInterface::retrieveConditionAny` would be the method to use and the timestamp could be `mo->getValidity().getMin()`, which now is still the time of creating an object, but soon it will be SOR in case of integrated plots and the cycle start in case of moving windows.

Reply:
Thanks a lot for pointing us to this. I modified the code using the methods you suggested.